administrator/language/pl-PL/com_scheduler.sys.ini

```
@@ -10,7 +10,7 @@ COM_SCHEDULER_MANAGER_TASK="Menedżer zadania"
 COM_SCHEDULER_MANAGER_TASK_EDIT="Edytuj zadanie"
 COM_SCHEDULER_MANAGER_TASK_NEW="Nowe zadanie"
 COM_SCHEDULER_MANAGER_TASKS="Menedżer zadań"
-COM_SCHEDULER_MSG_MANAGE_NO_TASK_PLUGINS="Nie ma typów zadań pasujących do twojego zapytania!"
+COM_SCHEDULER_MSG_MANAGE_NO_TASK_PLUGINS="Nie ma typów zadań pasujących do Twojego zapytania!"
```

Poprawić trzeba wszędzie tam, gdzie mamy użyte wielkie litery, nie ma żadnego powodu, by tego typu zaimki osobowe pisać wielką literą. Komunikaty, etykiety nie są osobistymi wypowiedziami człowieka do człowieka, to system komunikuje.
```suggestion
COM_SCHEDULER_MSG_MANAGE_NO_TASK_PLUGINS="Nie ma typów zadań pasujących do twojego zapytania!"
```